### PR TITLE
PROD-382/add-streaks

### PR DESCRIPTION
**Pull request by danijel-radomiljac**
Preview video: 

https://github.com/user-attachments/assets/2cdc3c78-6754-49e8-a4ff-6c175e14fe22

@marvinmarnold @Yihwan Since we are querying longest streak in a new way, can we delete a Streak table from database?

### CODE REVIEW

File: app/queries/home.ts [hunk 1]
```diff
@@ -375,3 +381,49 @@ async function queryUserStatistics(userId: string): Promise<UserStatistics> {
...
+
+export async function getUsersLongestStreak(): Promise<number> {
+  const payload = await authGuard();
+
+  const longestStreak = await queryUsersLongestStreak(payload.sub);
+
+  return longestStreak;
+}
+
+async function queryUsersLongestStreak(userId: string): Promise<number> {
```
Comment (marvinmarnold) [inline app/queries/home.ts:393]: Can you add tests. See reference test implementation: https://github.com/gator-labs/chomp/pull/579

**Review by marvinmarnold — CHANGES_REQUESTED** [review]

File: app/queries/home.ts (hunk 1 above)
Comment (danijel-radomiljac) [reply app/queries/home.ts:393]: I will wait for that pr to be merged first. Because there are some test setup changes.

Comment (yash431garg) [reply app/queries/home.ts:393]: Will try to merge it asap.

Comment (marvinmarnold) [reply app/queries/home.ts:393]: I'm OK to drop Streak from DB if not being used.

**Review by gatorthana — APPROVED** [review]
Looks good to me!

**Pull request merged**